packages/react-components/react-card/src/components/Card/useCard.ts

```diff
@@ -1,8 +1,38 @@
 import * as React from 'react';
 import { getNativeElementProps } from '@fluentui/react-utilities';
-import type { CardProps, CardState } from './Card.types';
 import { useFocusableGroup } from '@fluentui/react-tabster';
 
+import type { CardProps, CardRefElement, CardState } from './Card.types';
+import { useCardSelectable } from './useCardSelectable';
+
+const focusMap = {
+  off: undefined,
+  'no-tab': 'limited-trap-focus',
+  'tab-exit': 'limited',
+  'tab-only': 'unlimited',
+} as const;
```

Comment:
Is this needed? You removed it from the other files.

```suggestion
};
```